### ISSUE TITLE
Still some issue : share / embed photo and videos with SEO on and off not full links

### ISSUE THREAD

**Issue by spirogg** [MacWarrior/clipbucket-5.0#17]
not complete links for share / embed photos and videos see photos

- Just FYI :)
```
Html
<a href='/item/photos/13/4RYN66935O4A/Minecraft-Wallpaper-10' target='_blank'><img src='/plugins/cb_server_thumb/timthumb.php?src=152745017521655a_l.jpg&directory=photos/2018/05/27/&type=photos&h=505.546875&w=900&zc=1' title='Minecraft-Wallpaper-10' alt='Minecraft-Wallpaper-10 TV Clipz' /></a>
Forum
[URL=/item/photos/13/4RYN66935O4A/Minecraft-Wallpaper-10][IMG]/plugins/cb_server_thumb/timthumb.php?src=152745017521655a_l.jpg&directory=photos/2018/05/27/&type=photos&h=505.546875&w=900&zc=1[/IMG][/URL]
Email
/item/photos/13/4RYN66935O4A/Minecraft-Wallpaper-10
Direct
/plugins/cb_server_thumb/timthumb.php?src=152745017521655a_l.jpg&directory=photos/2018/05/27/&type=photos&h=505.546875&w=900&zc=1
```
SEO ON. 

![photos seo on](https://user-images.githubusercontent.com/17736688/40590994-c992dfae-61ce-11e8-9803-0258b1efe71f.PNG)
![video seo on](https://user-images.githubusercontent.com/17736688/40590995-cc5bf126-61ce-11e8-9bc5-e2eb7e1b23f1.PNG)

**Comment by spirogg** [comment]
-also with SEO OFF
-
![photos seo off](https://user-images.githubusercontent.com/17736688/40591093-5d7ba984-61d0-11e8-8d6d-7474b302fd90.PNG)
![video seo off](https://user-images.githubusercontent.com/17736688/40591095-602dc5a4-61d0-11e8-96b4-67f82739372d.PNG)

**Comment by MacWarrior** [comment]
My bad, I did not see it when I switched all URLs to relative URLs

**Comment by MacWarrior** [comment]
Hi,
Please update to this commit : 216fc15586b356d2c1acd281658b7e9f3e5105e5

**Comment by spirogg** [comment]
`<object width="300" height="250"><param name="movie" value="/player/embed_player.php?vid=1036"></param><param name="allowFullScreen" value="true"></param><param name="allowscriptaccess" value="always"></param><embed src="/player/embed_player.php?vid=1036"type="application/x-shockwave-flash" allowscriptaccess="always" allowfullscreen="true" width="300" height="250"></embed></object>`

**Comment by spirogg** [comment]
- also photos still have some issues

```
HTML

<a href='https://mydomain.com/view_item.php?item=8W68D8H8KMUB&type=photos&collection=17' target='_blank'><img src='/files/photos/2018/06/02/1527921028f43dc0.png' title='bugs bunny dunking' alt='bugs bunny dunking TV Clipz' /></a> not show image <img src'=


FORUM

[URL=https://mydomain.com/view_item.php?item=8W68D8H8KMUB&type=photos&collection=17][IMG]/files/photos/2018/06/02/1527921028f43dc0.png[/IMG][/URL] not show image [IMG]


EMAIL

https://mydomain.com/view_item.php?item=8W68D8H8KMUB&type=photos&collection=17


DIRECT

/files/photos/2018/06/02/1527921028f43dc0.png  not working


```

**Comment by MacWarrior** [comment]
I have another bug for Direct URL : 
![capture](https://user-images.githubusercontent.com/200590/40891637-a774daec-6789-11e8-8250-afa354bcaecd.PNG)
This must be a Array to string conversion : will investigate on this

**Comment by MacWarrior** [comment]
My video embed code seems correct : 
![capture](https://user-images.githubusercontent.com/200590/40891666-4d6084c4-678a-11e8-8384-88f25e6b7341.PNG)
I got in on FO video page > Sharing tab

**Comment by spirogg** [comment]
There is 2 options embed code and Video Embed Object. The video embed object not working. 

- Reg embed code working. For me too.

**Comment by MacWarrior** [comment]
Hi, 
Fixed in this commit : d8535e6fa35cd0ed4737854bcaa98bdcc49a3f3a
